### PR TITLE
chore: add typescript definition

### PR DESCRIPTION
**Pull request by achingbrain**

### CODE REVIEW

**Comment by rvagg** [comment]
ah, found the source! better merge this I suppose since it's already been published. This lgtm

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipld/interface-ipld-format.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (rvagg) [inline package.json]: better fix this, `"(Apache-2.0 AND MIT)"`

**Comment by achingbrain** [comment]
> better merge this I suppose since it's already been published

I pre-emptively published this yesterday to do a land-grab of the npm name as I noticed nothing had been published there.

So far it's only being used by [this series of PRs](https://github.com/ipld/js-ipld/pull/297) and my outstanding PR to js-IPFS updating all the types so we should be free to make changes as required.

File: package.json (hunk 1 above)
Comment (achingbrain) [inline package.json]:
```suggestion
  "license": "(Apache-2.0 AND MIT)",
```

Comment (achingbrain) [reply package.json]: Done!

**Review by rvagg — APPROVED** [review]

**Review by vmx — APPROVED** [review]

**Pull request merged**